SU-SWS/sulgryphon-nextjs
# SUL23-43 Style the basic page card and list items

# READY FOR REVIEW

# Summary
- Updated to NextJS 13
- Style the basic page card and list displays

# Need Review By (Date)
- 

# Urgency
- 

# Steps to Test
1. checkout this branch
2. `yarn`
3. create some basic pages in drupal and populate the metadata fields
4. create a basic page and use the teaser paragraph type to choose the pages you created earlier.
5. verify the teasers look as expected.


<a href="https://gitpod.io/#https://github.com/SU-SWS/sulgryphon-nextjs/pull/14"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
